backend/dal/Helpers/Extensions/EntityEntryExtensions.cs

```diff
@@ -44,15 +33,10 @@ public static void UndoAppCreatedEdits(this EntityEntry entry)
         /// <param name="entry"></param>
         public static void UndoDbCreatedEdits(this EntityEntry entry)
         {
-            if (entry.Entity is IDisableBaseAppEntity entity)
+            if (entry.Entity is IBaseAppEntity)
             {
-                var oCreatedOn = (DateTime)entry.OriginalValues[nameof(entity.DbCreateTimestamp)];
-                if (!oCreatedOn.Equals(entity.DbCreateTimestamp))
-                    entity.DbCreateTimestamp = oCreatedOn;
-
-                var oCreatedBy = (string)entry.OriginalValues[nameof(entity.DbCreateUserid)];
-                if (String.IsNullOrWhiteSpace(entity.DbCreateUserid) || !oCreatedBy.Equals(entity.DbCreateUserid))
-                    entity.DbCreateUserid = oCreatedBy;
+                entry.Property(nameof(IBaseAppEntity.DbCreateTimestamp)).IsModified = false;
+                entry.Property(nameof(IBaseAppEntity.DbCreateUserid)).IsModified = false;
```

Comment:
Same as above, fixed entity type and removed tracking of unnecessary entity change.